opennms-base-assembly/src/main/filtered/etc/xml-datacollection-config.xml

```diff
@@ -0,0 +1,15 @@
+<xml-datacollection-config rrdRepository="${install.share.dir}/rrd/snmp/"
+                           xmlns="http://xmlns.opennms.org/xsd/config/xml-datacollection">
+    <xml-collection name="elasticsearch-stats">
+        <rrd step="300">
+            <rra>RRA:AVERAGE:0.5:1:2016</rra>
+            <rra>RRA:AVERAGE:0.5:12:1488</rra>
+            <rra>RRA:AVERAGE:0.5:288:366</rra>
+            <rra>RRA:MAX:0.5:288:366</rra>
+            <rra>RRA:MIN:0.5:288:366</rra>
+        </rrd>
+        <xml-source url="http://localhost:9200/_cluster/stats">
```

Comment:
"localhost" should be changed to "{ipaddr}" so that the IP address being collected is substituted for the value in the URL.